shared/actions/chat2/index.js

```diff
@@ -1257,6 +1258,7 @@ const previewConversationFindExisting = (
     users = I.Set(participants)
       .subtract([you])
       .toArray()
+    setUsers = Saga.put(Chat2Gen.createSetPendingConversationUsers({fromSearch: false, users}))
```

Comment:
Here's where I set the users to avoid the pending conversation having 'New conversation'.

Ideally, we'd be able to handle this like updating the pending mode, but I don't see a clean way of doing that yet. Perhaps after the follow-up PR to split the actions it'll become easier.